Completed the UI, including adding in pages for "About" and "Contact.…

radiantdinosaurs opened the pull request:
…" Since the UI used some jQuery, I enabled the jQuery environment variable in linting document.

radiantdinosaurs committed:
Completed the UI, including adding in pages for "About" and "Contact." Since the UI used some jQuery, I enabled the jQuery environment variable in linting document.

andrewvora reviewed (changes requested):
Looks good, but let's consolidate the header element into a single layout that gets included in every page that needs it. Like we discussed, use the `if`s to decide whether to display the green element or the white one

radiantdinosaurs committed:
Fixed requested changes.

radiantdinosaurs committed:
Fixed requested changes.

radiantdinosaurs committed:
Updated package and hammered out the UI.
Completely finished styling the UI (for now). Made some minor fixes, and updated packages for security.
